tracing-attributes/src/lib.rs

```diff
@@ -132,11 +132,14 @@ mod expand;
 /// Setting the level for the generated span:
 /// ```
 /// # use tracing_attributes::instrument;
-/// #[instrument(level = "debug")]
+/// #[instrument(level = Level::DEBUG)]
 /// pub fn my_function() {
 ///     // ...
 /// }
 /// ```
+/// Levels can be specifide either with `Level` constants, literal strings
+/// (`"debug", etc) or numerically (1 - 5).
```

Comment:
maybe worth indicating how numbers are mapped to `Level`s --- maybe with a table, or just mentioning that 5 corresponds to the `TRACE` level? we could link to the docs here: https://docs.rs/tracing-core/latest/tracing_core/struct.Level.html#comparing-levels